core/arch/aarch64/proc.c

```diff
@@ -112,6 +112,27 @@ proc_num_simd_sse_avx_saved(void)
     return 0;
 }
 
+int
+proc_xstate_area_kmask_offs(void)
+{
+    ASSERT_NOT_IMPLEMENTED(false);
```

Comment:
I find this a little confusing: normally this assert means there is work to be done here but nobody has done it yet.  In this case this simply doesn't apply to this architecture.  I think your assert is trying to say that it's an error to call this function, maybe `ASSERT_NOT_REACHED` does a better job of that if callers should never call this.  In either case, a comment would probably be a good idea to make it clear this is just not relevant for this architecture.

Ditto below.

Reply:
Done.